libs/portal/shared/features/form/ui/month-year-field.tsx

```
@@ -0,0 +1,224 @@
+                </FormLabel>
+              </div>
+
+              <div className="flex gap-x-3 w-full items-center">
+                <img src={calendarIcon} alt="Calendar" className="size-6" />
+
+                {/* Month Combobox */}
+                <Popover open={monthOpen} onOpenChange={setMonthOpen}>
+                  <PopoverTrigger asChild>
+                    <Button
+                      variant="outline"
+                      role="combobox"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<select>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)